src/index.ts

```diff
@@ -29,6 +29,21 @@ app.set('view engine', 'jsx')
 app.engine('jsx', require('express-react-views').createEngine())
 app.set('trust proxy', config.TRUST_PROXY_IPS)
 
+app.disable('x-powered-by')
+app.use(function (_req, res, next) {
+  res.setHeader(
+    'Content-Security-Policy',
+    "default-src 'self' tarpisto.cdn.tko-aly.fi; img-src 'self' tarpisto.cdn.tko-aly.fi; media-src 'self' tarpisto.cdn.tko-aly.fi; script-src 'self' tarpisto.cdn.tko-aly.fi;"
```

Comment:
Not sure if cdn domain needs to be allowed at all 🤔 I think we only use it to serve the documents right now and there's no plans to start serving JS/CSS assets from there.

Reply:
Dropped CDN out.